api/handlers/containerhandler.py

```diff
@@ -111,7 +112,13 @@ def get(self, cont_name, **kwargs):
         # build and insert file paths if they are requested
         if self.is_true('paths'):
```

Comment:
I don't believe this flag is used anywhere. I will confirm and hopefully we can just remove this path.

Reply:
Following up, support of the `paths` query param can be removed.